lib/src/react_client/synthetic_event_wrappers.dart

```diff
@@ -8,8 +8,12 @@ import 'dart:html';
 import 'package:js/js.dart';
 
 @JS()
-@anonymous
 class SyntheticEvent {
```

Comment:
I'm fine doing this in a separate PR, but at some point we should add doc comments on these classes and fields (there are some we can transfer over from the old classes).

Reply:
I can add it to this one, that's easy!